config/passport.js

```diff
@@ -0,0 +1,62 @@
+const passport = require( 'passport' )
+const User = require( '../models/users' )
+
+const { Strategy, ExtractJwt } = require( 'passport-jwt' )
+
+const LocalStrategy = require( 'passport-local' )
+
+const localOptions = {
```

Comment:
This should be named using standard naming conventions for constants: `LOCAL_OPTIONS` (and updated below)